dynawo/sources/Modeler/Common/DYNSubModel.cpp

```diff
@@ -670,6 +671,32 @@ SubModel::defineParameters(const bool isInitParam) {
   addParameters(parameters, isInitParam);
 }
 
+void
+SubModel::printInitValues(const std::string& directory) {
+  const string& fileName = absolute("dumpInitValues-" + name() + ".txt", directory);
+
+  std::ofstream file;
+  file.open(fileName.c_str());
+
+  printInitValuesVariables(file);
+  printInitValuesParameters(file);
+
+  file.close();
+}
+
+void
+SubModel::printInitValuesParameters(std::ofstream& fstream) {
+  fstream << " ====== PARAMETERS VALUES ======\n";
+  for (boost::unordered_map<std::string, ParameterModeler>::const_iterator it = parametersDynamic_.begin(); it != parametersDynamic_.end(); ++it) {
```

Comment:
This dumps the parameter in a non determinism order

Reply:
Since the map is always filled in the same order, couldn't we assume it is displayed in the same order ?

Reply:
This order depends on the OS whereas the file should always be the same independently from the environment of its generation

Reply:
OK